packages/integration-karma/test/wire/reactive-params.spec.js

```diff
@@ -0,0 +1,20 @@
+import { createElement } from 'lwc';
+
+import CascadeWiredProps from 'x/cascadeWiredProps';
+
+describe('@wire reactive parameters', () => {
+    if (process.env.COMPAT !== true) {
```

Comment:
this functionality does not works in compat due the fact that the wire-service can't extract the values of an ie11 proxy object.

Reply:
This should be added as a comment inline with test code, otherwise, we will forget in the future.